validation/src/test/scala/hmda/validation/rules/lar/validity/V385Spec.scala

```diff
@@ -0,0 +1,40 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V385Spec extends LarEditCheckSpec {
+
+  val actionTaken = List(3, 7)
+
+  property("If agency code is not 1 or actionTaken is not 3 or 7, lar must succeed") {
+    forAll(larGen) { lar =>
+      whenever(lar.agencyCode != 1 || !actionTaken.contains(lar.actionTakenType)) {
+        lar.mustPass
+      }
+    }
+  }
+
+  val actionTakenGen: Gen[Int] = Gen.oneOf(actionTaken)
+  val validDenialGen: Gen[Int] = Gen.choose(1, 9)
+
+  property("If the agency code and actionTaken are correct and denial reasons are correct, lar must pass") {
```

Comment:
Slightly nitpicky: Technically, agency code and actionTaken are correct even if they aren't 1 and 3/7 respectively--there are other valid values. But 1 and 3/7 are the only values that trigger the condition of this edit.

A more accurate property name would be something like "Succeeds if agency code = 1, actionTaken = 3,7, and denial reasons are valid"